scalyr_agent/monitor_utils/k8s.py

```diff
@@ -86,7 +85,7 @@
     },
 }
 
-def cache( config ):
+def cache(global_config):
```

Comment:
Especially for  newbies, it's hard to keep track of all the `config`s floating around -- global, monitor-specific or cache?!  Renamed this to `global_config` for clarity.